README.md

```diff
@@ -438,6 +438,27 @@ Creating a search key that will only search over the body field.
 {'source_engines': ['source-engine-1', 'source-engine-2'], 'type': 'meta', 'name': 'my-meta-engine'}
 ```
 
+### Search the API logs
+
+```python
+>>> client.search_api_logs('my-meta-engine', {
+  "filters": {
+    "date": {
+      "from": "2020-03-30T00:00:00+00:00",
+      "to": "2020-03-31T00:00:00+00:00"
+    },
+    "status": "429",
+  }
+})
+{'results': [],
```

Comment:
Could you update the formatting here to match the formatting of other responses in this README?

Reply:
Sure, I don't recall if that was the copied/pasted output or where that formatting came from. Not be facetious at all, but I'm not sure _which_ format you prefer. I see examples of "pretty" formatted output, one-liners, and one-liners with elided output:

<img width="460" alt="Screen Shot 2020-07-29 at 9 39 18 AM" src="https://user-images.githubusercontent.com/24574/88827885-5aa47300-d17f-11ea-8cbf-4f82d1a3f49e.png">

Reply:
You have a great point. How about "pretty"?